Library/Formula/terminal-notifier.rb

```diff
@@ -27,6 +27,6 @@ def install
   end
 
   test do
-    system "#{bin}/terminal-notifier", "-message", "Hello World!"
+    system "#{bin}/terminal-notifier", "-help"
```

Comment:
Please do `if MacOS.version < :yosemite` on the end of this line to avoid the error.